trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/validate-facility-id-query.ts

```diff
@@ -1,28 +1,18 @@
-import { Request } from 'express';
-import { asString, isNonEmptyString } from '@ukef/dtfs2-common';
+import { isNonEmptyString } from '@ukef/dtfs2-common';
 import { ErrorSummaryViewModel } from '../../../types/view-models';
 import { REGEX } from '../../../constants';
 
-export const validateFacilityIdQuery = (
-  req: Request,
-): {
-  validatedFacilityIdQuery: string | undefined;
-  facilityIdQueryError: ErrorSummaryViewModel | undefined;
-} => {
-  const { facilityIdQuery } = req.query;
-  const { originalUrl } = req;
-  const facilityIdQueryAsString = facilityIdQuery ? asString(facilityIdQuery, 'facilityIdQuery') : undefined;
-  let facilityIdQueryError: ErrorSummaryViewModel | undefined;
-
+export const validateFacilityIdQuery = (facilityIdQuery: string | undefined, originalUrl: string): ErrorSummaryViewModel | undefined => {
```

Comment:
The use of `originalUrl` in this function still feels a bit redundant to me. The `originalUrl.includes('?') && originalUrl.includes('facilityIdQuery')` should be captured by `req.query.facilityIdQuery === undefined` because of the way that the request object is generated, so checking the actual `url` shouldn't be required

Reply:
It's not redundant because if they click filter without typing anything the query param is tacked onto the url but when you read the value it is undefined and in that case we want to display an error message, but not in the case where it is undefined because there is no query param at all because they didn't just click the filter button.